data/virt_autotest/kubevirt_tests/full-tests.conf

```diff
@@ -97,11 +97,11 @@ GINKGO_SKIP=test_id:4099|test_id:4137|test_id:4139|test_id:6535
 [operator_test]
 GINKGO_FOCUS=operator_test.go
 GINKGO_SKIP=test_id:3153
-EXTRA_OPT=-previous-release-registry=PREVIOUS_RELEASE_REGISTRY -previous-release-tag=PREVIOUS_RELEASE_TAG -skip-shasums-check
```

Comment:
Just to double confirm, without specifying previous-release-xx, for suse downstream test, will suse's downstream ones be automatically detected?

Reply:
These parameters are moved to server test module as common part for all cases.